packages/dominus-settings/settingsCapitals.js

```diff
@@ -29,7 +20,16 @@ _gs.capitals = function(gameId, path) {
       glass: 80
     }
 
+    settings.startingGarrison = {
+      footmen: 24,
+      archers: 24,
+      pikemen: 24,
+      cavalry: 0,
+      catapults: 0
+    }    
+
     settings.incomeInterval = 1000 * 60 * 10;
+    settings.villagePercentageIncome = 0.12;
```

Comment:
Reverted with https://github.com/dan335/dominus/pull/2/commits/3ba0381f06773c0d8a27529b3583c123feea53e2 . Game running for 20 minutes now, no errors.